backend/database/wrapper/field_wrapper.py

```diff
@@ -149,23 +158,39 @@ def get_explanations(dynamic_field_list) -> (Dict[str, FieldExplanation], List[F
                          'Total time being the player closest to the positional center of their team.'),
         FieldExplanation('time_furthest_from_team_center',
                          'Total time being the player furthest from the positional center of their team.'),
+        FieldExplanation('time_in_corner',
```

Comment:
We may want to use the word field instead of pitch.  (US are more likely to only understand field)

I think that is used elsewhere in explanations

Reply:
Good catch. Terminology should be consistent. Will use 'field'.